tests/unit/memoizers/meta-tags-context-memoizer-test.php

```diff
@@ -0,0 +1,325 @@
+<?php
+
+namespace Yoast\WP\SEO\Tests\Unit\Memoizers;
+
+use Brain\Monkey;
+use Mockery;
+use WP_Post;
+use Yoast\WP\SEO\Context\Meta_Tags_Context;
+use Yoast\WP\SEO\Helpers\Blocks_Helper;
+use Yoast\WP\SEO\Helpers\Current_Page_Helper;
+use Yoast\WP\SEO\Memoizers\Presentation_Memoizer;
+use Yoast\WP\SEO\Repositories\Indexable_Repository;
+use Yoast\WP\SEO\Tests\Unit\Doubles\Context\Meta_Tags_Context_Mock;
+use Yoast\WP\SEO\Tests\Unit\Doubles\Memoizers\Meta_Tags_Context_Memoizer_Double;
+use Yoast\WP\SEO\Tests\Unit\Doubles\Models\Indexable_Mock;
+use Yoast\WP\SEO\Tests\Unit\TestCase;
+
+/**
+ * Class Meta_Tags_Context_Memoizer_Test.
+ *
+ * @coversDefaultClass \Yoast\WP\SEO\Memoizers\Meta_Tags_Context_Memoizer
+ *
```

Comment:
```suggestion
 * @covers \Yoast\WP\SEO\Memoizers\Meta_Tags_Context_Memoizer
```

Reply:
We don't want to use @covers for classes because it means it covers the entire class. If new functions are added to the class the would be falsely marked as tested in the code coverage.